### PR TITLE
Minor Correction on Validation Cases

### PR DESCRIPTION
**Pull request by yfche**
Close. #217 

## Reason
Minor changes for val-2b and val-2c

## Design
See changed files

### CODE REVIEW

File: test/tests/val-2c/comparison_val-2c.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def sample_solution(time_stamps, time_solution, value_solution):
 else:                                  # if in test folder
     csv_folder = "./gold/val-2c_delay_csv.csv"
 tmap8_prediction_delay = pd.read_csv(csv_folder)
-tmap8_time_delay = tmap8_prediction['time']
+tmap8_time_delay = tmap8_prediction_delay['time']
```
Comment (simopier) [inline test/tests/val-2c/comparison_val-2c.py:99]: Good catch!

File: test/tests/val-2b/gold/experimental_data.csv [hunk 2]
```diff
@@ -114,19 +114,19 @@ temperature (C),flux (atoms/mm^2/s x 10^10)
...
-748.2738531,0.014894817
-746.852224,0.003144617
-742.1034776,0.027094247
-740.2129636,-0.020131166
 738.309347,0.015484433
-730.7192142,0.004099233
-661.8474159,0.44604431
-664.6944176,0.445875848
\ No newline at end of file
+740.2129636,-0.020131166
```
Comment (simopier) [inline test/tests/val-2b/gold/experimental_data.csv:129]:
```suggestion
740.2129636,0
```
The values should be positive, this is an artifact of the plot digitization.

**Review by simopier — CHANGES_REQUESTED** [review]
Just some minor comments, thank you for this fix!

**Comment by yfche** [comment]
@simopier Is there a way to get around the "Test Heavy" error? I ran into it with the previous push, too.
Also, just being curious, is there a reason why the checks are taking so long today?

**Comment by simopier** [comment]
The heavy tests were recently added, and the failures are unrelated to this change. I have a fix about to be pushed. 

I think it's because I first needed to activate the tests, and then it might have been slow because of the HPC outage.

**Review by simopier — APPROVED** [review]
Thank you!

**Pull request merged**